### DISCUSSION TITLE
Bypassing automatic mopping

### DISCUSSION THREAD

**Discussion by incline02** [mrbungle64/ecovacs-deebot.js#302] [Ideas]
Hi mrbungle64

Currently on some deebot, if the user installs the reservoir+mop, the robot switches to mopping mode instantly and uses the water. there's no out-of-the-box way to bypass that. I was wondering if there was a software way and if it could be on your plugin's end? Something like overriding mopping mode to hoovering mode and/or switching to a 0 water-spillage (the current minimum is 1, which is using water, up to 4).

My use case is the following: The robot turns on 3 times a week, including Sunday. On Sundays, I would like it to use  the water at level 4. The rest of the week, different parameters. In short, my use case is: I would like to automate different routines via scenes or else, whilst not having to manually take out the reservoir each time I just want to hoover.

Is it something you have/can look into?

in advance, thanks

**Comment by mrbungle64** [comment]
@incline02 

What model are you using and are you already using ioBroker?

**Comment by incline02** [comment]
Deebot N8+ Pro

I’m not using ioBroker, only your plugin to homebridge

**Reply by mrbungle64** [reply]
@incline02 

I don't think that it's possible with this model.
The X1 has an "mopping only" option, but there's no "vacuum cleaning only" option ...

I also don't think that makes sense at all as long as the mop is installed...
